Description: Fix PSGI plugin building with defining HAS_BOOL only on Apple
Origin: upstream, http://projects.unbit.it/hg/uwsgi/rev/5c86defc1864
Last-Update: 2011-05-01

Index: uwsgi/plugins/psgi/psgi.h
===================================================================
--- uwsgi.orig/plugins/psgi/psgi.h	2011-04-29 04:26:50.000000000 +0300
+++ uwsgi/plugins/psgi/psgi.h	2011-04-29 04:27:15.000000000 +0300
@@ -1,7 +1,9 @@
 #undef __USE_GNU
 #include "../../uwsgi.h"
 
+#ifdef __APPLE__
 #define HAS_BOOL 1
+#endif
 #include <EXTERN.h>
 #include <perl.h>
 #include "XSUB.h"
